commcare_connect/form_receiver/views.py

```diff
@@ -8,7 +9,8 @@
 
 class FormReceiver(APIView):
     parser_classes = [parsers.JSONParser]
-    permission_classes = [permissions.IsAuthenticated]
+    authentication_classes = [OAuth2Authentication]
+    permission_classes = [TokenHasReadWriteScope]
```

Comment:
The token that we get from the client credentials flow comes with `scope: "read write"` by default. I checked this by manually doing the flow. Data forwarding is working using the above changes and the data is getting forwarded to CCC. The tests could be failing as we have changed the permission on this API from IsAuthenticated to TokenHasReadWriteScope, this could be fixed by including using the APIClient fixture and specifying credentials to have a Authorization with Bearer token.